### DISCUSSION TITLE
form php plugin

### DISCUSSION THREAD

**Discussion by wezetel** [trobfab/fabrik#514] [General]
Quick question: in the status of plugin the comment for the plugin states '(onLoad, onBefore) working with default settings'. Does this mean these are the tested cases and the others won't work or are simply not tested?

Background: I have this plugin on my production site (F3) started with onAfterProcess (changes values in another table). With F4 this also worked some time ago, but now not any longer (not sure which of my actions resulted in this behaviour). This is not an issue, as onBeforeProcess works. As this also is only triggered after validations have been performed, I don't mind, if the form data or the other table are changed first.

**Comment by trob** [comment]
It means these setups were tested and working (at time of writing).

If some part of a plugin is showing issues it should be mentioned explicitely.

**Reply by wezetel** [reply]
Thanks. That answered my question!
